app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java

```diff
@@ -130,11 +127,7 @@ public void updateFromItem(final InfoItem infoItem,
 
         itemView.setOnLongClickListener(view -> {
             if (!AndroidTvUtils.isTv(itemBuilder.getContext())) {
-                ClipboardManager clipboardManager = (ClipboardManager) itemBuilder.getContext()
-                        .getSystemService(Context.CLIPBOARD_SERVICE);
-                clipboardManager.setPrimaryClip(ClipData.newPlainText(null, commentText));
-                Toast.makeText(itemBuilder.getContext(), R.string.msg_copied, Toast.LENGTH_SHORT)
-                        .show();
+                ShareUtils.copyToClipboard(itemBuilder.getContext(), commentText);
             } else {
                 openCommentAuthor(item);
             }
```

Comment:
Even though this is unrelated from your changes, could you invert the two branches of the if, while also removing the `!` ?
```suggestion
            if (AndroidTvUtils.isTv(itemBuilder.getContext())) {
                openCommentAuthor(item);
            } else {
                ShareUtils.copyToClipboard(itemBuilder.getContext(), commentText);
            }
```

Reply:
I made that code, the logic behind it was that non-TV is more common